tox.ini

```diff
@@ -22,31 +25,16 @@ passenv =
   CHARM_BUILD_DIR
   MODEL_SETTINGS
 
-[testenv:fmt]
+[testenv:format]
```

Comment:
I'm making this change according to the upcoming [OP061 recommendation](https://docs.google.com/document/d/1GfOTT1Ir-pLAbILUrI4GS9T8AAI8Ni8gpF1Mh67Wx3E/edit?tab=t.0#heading=h.kavgppauvkj5).